components/style/gecko/non_ts_pseudo_class_list.rs

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+ * This file contains a helper macro includes all supported non-tree-structural
+ * pseudo-classes.
+ *
+ * This file is NOT INTENDED to be compiled as a standalone module.
```

Comment:
This is not autogenerated right? Please note it in this comment (potentially keeping the `FIXME` that you removed from the other file).